docs/requirements.txt

```diff
@@ -9,3 +12,6 @@ pytz
 psutil
 pyparsing
 GSI
+M2Crypto
```

Comment:
Can we removed GSI?

Reply:
Yes, GSI dropped, though there is an issue with installing m2crypto 0.33 (released an hour ago) on RTD, so fixed to 0.32

Reply:
well, m2crypto 0.33 also doesnt work on travis